lib/winapi/winioctl.h

```diff
@@ -0,0 +1,18 @@
+#ifndef __WINIOCTL_H__
+#define __WINIOCTL_H__
+
+#include <xboxkrnl/xboxdef.h>
+
+typedef struct _FILE_OBJECTID_BUFFER {
+  BYTE  ObjectId[16];
+  union {
+    struct {
+      BYTE BirthVolumeId[16];
+      BYTE BirthObjectId[16];
+      BYTE DomainId[16];
+    } DUMMYSTRUCTNAME;
+    BYTE ExtendedInfo[48];
+  } DUMMYUNIONNAME;
+} FILE_OBJECTID_BUFFER, *PFILE_OBJECTID_BUFFER;
```

Comment:
Is this even used? Why is this part of this PR?

Reply:
You're right, it's not anymore. It was part of my attempt to use `NtFsControlFile` to determine if the two paths referenced the same underlying file and I failed to clean it out when I abandoned that approach.

Temporarily adding back in that code in the hopes that there's a silly mistake that you can point out. The issue I'm seeing is that the NFCF call returns STATUS_INVALID_DEVICE_REQUEST

Reply:
Fully removed, it seems likely that FATX doesn't support objectids at all.